### PR TITLE
[4.1] Joomla Update Component update site

### PR DESCRIPTION
**Pull request by brianteeman**
### Summary of Changes
Simple PR to change the name of the update site for the Joomla Update Component.

It doesn't really make sense to have Update Site in the name

### Testing Instructions
Update using the prebuilt updates
Clean install using the prebuilt full package

### Actual result BEFORE applying this Pull Request
image

### Expected result AFTER applying this Pull Request
image

### Documentation Changes Required

none

### CODE REVIEW

File: administrator/components/com_admin/sql/updates/mysql/4.1.3-2022-04-08.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+UPDATE `#__update_sites`
+   SET `name` = 'Joomla! Update Component'
+ WHERE `name`= 'Joomla! Update Component Update Site';
```
Comment (Quy) [inline administrator/components/com_admin/sql/updates/mysql/4.1.3-2022-04-08.sql]:
```suggestion
 WHERE `name` = 'Joomla! Update Component Update Site';
```

File: administrator/components/com_admin/sql/updates/postgresql/4.1.3-2022-04-08.sql [hunk 2]
```diff
@@ -0,0 +1,3 @@
+UPDATE "#__update_sites"
+   SET "name" = 'Joomla! Update Component'
+ WHERE "name"= 'Joomla! Update Component Update Site';
```
Comment (Quy) [inline administrator/components/com_admin/sql/updates/postgresql/4.1.3-2022-04-08.sql]:
```suggestion
 WHERE "name" = 'Joomla! Update Component Update Site';
```

**Review by richard67 — APPROVED** [review]

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 74cc1a2bccdb1f59eb6b1d271c892b9c5a7a8419<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37505">issues.joomla.org/tracker/joomla-cms/37505</a>.</sub>

**Review by wilsonge — APPROVED** [review]

**Comment by chmst** [comment]
I have tested this item :white_check_mark: successfully on 74cc1a2bccdb1f59eb6b1d271c892b9c5a7a8419<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37505">issues.joomla.org/tracker/joomla-cms/37505</a>.</sub>

**Comment by chmst** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37505">issues.joomla.org/tracker/joomla-cms/37505</a>.</sub>

**Comment by laoneo** [comment]
Thanks!

**Pull request merged**